tree/master/cloud/src/Helper Utilities/Microsoft.Legal.MatterCenter.HelperUtilities/Microsoft.Legal.MatterCenter.UpdateAppConfig/UpdateConfig.cs

```diff
@@ -332,7 +332,13 @@ public static bool UpdateXML(Dictionary<string, string> excelInput)
                                         {
                                             case 2:
                                                 appDomain.InnerText = excelInput[ConstantStrings.AZURE_UI_SITE_URL];
-                                                break;                                            
+                                                break;
+                                            case 3:
+                                                if (appDomain.InnerText.ToLower().Contains("sharepoint"))

```

Comment:
@AISDUG  updating site url and tenant url is same for both outlook and office.  remove 4th one and use 2nd entry for tenant url for both word and outlook app.